app/Resources/translations/messages.nl_NL.xliff

```diff
@@ -500,18 +517,38 @@ Er is een e-mail met activatiecode gestuurd naar het e-mailadres %email%. Volg d
       <trans-unit id="59ae5f416e832eeec457e8f73d11d196bcf9c6e5" resname="ss.second_factor.list.title">
         <source>ss.second_factor.list.title</source>
         <target>Overzicht tokens</target>
-        <jms:reference-file line="4">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/SecondFactor/list.html.twig</jms:reference-file>
+        <jms:reference-file line="3">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/SecondFactor/list.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="64cd3b12ab8e092173e5e99069d38afdaf811b16" resname="ss.second_factor.remote_vet.text.are_you_sure">
         <source>ss.second_factor.remote_vet.text.are_you_sure</source>
         <target>Weet je zeker dat je je identiteit wilt valideren met behulp van een externe dienst?</target>
         <jms:reference-file line="9">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/RemoteVetting/remoteVet.html.twig</jms:reference-file>
       </trans-unit>
+      <trans-unit id="789743faa71a92af3b0c7580952c3a47e62048ac" resname="ss.second_factor.remote_vet.text.does_it_match">
+        <source>ss.second_factor.remote_vet.text.does_it_match</source>
+        <target>Is deze informatie juist?</target>
+        <jms:reference-file line="9">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/RemoteVetting/validation.html.twig</jms:reference-file>
+      </trans-unit>
       <trans-unit id="460752b1bf195dc38fb4fcce77b3603c6de3b7fd" resname="ss.second_factor.remote_vet.title">
         <source>ss.second_factor.remote_vet.title</source>
         <target>Bevestig identiteit</target>
         <jms:reference-file line="4">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/RemoteVetting/remoteVet.html.twig</jms:reference-file>
       </trans-unit>
+      <trans-unit id="9c1a0c3ced900e4d33e110a5c2772ca68e85bf0f" resname="ss.second_factor.remote_vet_validation.title">
+        <source>ss.second_factor.remote_vet_validation.title</source>
+        <target>Controleer informatie</target>
+        <jms:reference-file line="4">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/RemoteVetting/validation.html.twig</jms:reference-file>
+      </trans-unit>
+      <trans-unit id="be7ea23287910e8499cd162adecc43ea8c072a20" resname="ss.second_factor.revoke.alert.remote_vetting_failed">
+        <source>ss.second_factor.revoke.alert.remote_vetting_failed</source>
+        <target>De identiteits informatie kon niet worden gevalideerd</target>
```

Comment:
Found a little typo: identiteits informatie is one word in Dutch: identiteitsinformatie

Same typo in `ss.second_factor.revoke.alert.remote_vetting_successful`

Reply:
Changed